src/app/app-modules/set-password/set-password.component.html

```
@@ -1,106 +1,100 @@
-<app-header [showRoles]="false"></app-header>
+<div class="support-wrapper">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)